geometry_msgs/Pose pose
bool linearMotion
---
bool success